Sprinkle of Yeezus/Settings.swift

```diff
@@ -11,52 +11,68 @@ import UIKit
 import UserNotifications
 
 class SettingsPage: UIViewController {
-    
 
-    @IBOutlet weak var InfoLabel: UILabel!
-    @IBOutlet weak var DescButton: UIButton!
-    @IBOutlet weak var DescLabel: UILabel!
-    @IBOutlet weak var Switch: UISwitch!
+    @IBOutlet private weak var InfoLabel: UILabel!
+    @IBOutlet private weak var DescButton: UIButton!
+    @IBOutlet private weak var DescLabel: UILabel!
+    @IBOutlet private weak var Switch: UISwitch!
+    @IBOutlet private weak var TimePicked: UIDatePicker!
     
-    var notificationsOn = Bool()
```

Comment:
I removed the `notificationsOn` variable and elected to just derive the value from the Switch instead. However, if you would like to still have a variable, I would recommend using a Computed Property:
```
var notificationsOn: Bool {
    return Switch.isOn
}
```

In this way, you don't ever need to set `notificationsOn`, it is just always equal to `Switch.isOn`.

Also, it's not recommended using `= Bool()`, it should either have a default value like `var notificationsOn = false` or be refactored (like using the Computer Property I showed above).